pkg/storage/store_test.go

```diff
@@ -102,3 +107,29 @@ func createTRBRequest(ctx context.Context, s *StoreTestSuite, createdBy string)
 
 	return createdRequest.ID
 }
+
+func getTestPrincipal(store *Store, userName string) *authentication.EUAPrincipal {
```

Comment:
Same comment as the one I made in `integration_test.go`

Reply:
Thanks Clay! Implemented